src/main/pg/max7456.h

```diff
@@ -27,6 +27,7 @@ typedef struct max7456Config_s {
     uint8_t clockConfig; // SPI clock based on device type and overclock state (MAX7456_CLOCK_CONFIG_xxxx)
     ioTag_t csTag;
     uint8_t spiDevice;
+    uint8_t preInitOPU;
```

Comment:
Not super happy with this - who's to say when the next non - MAX7456 device that needs OPU will be added to a board - we are setting us up to potentially having to do this for practically all devices, like has happened to half duplex for UARTS.

But I guess you can't fend this off before it's happening, and doing a more generic solution for just one device type is probably going over the top.

Reply:
TBH, it's beyond my imagination how we can make this generic. Do you want pre-init spec to be added to all devices now?

Reply:
I was more thinking in terms of defining a `csPin_t` struct containing the `ioTag_t` and `preInitOPU`, and then using this for all CS pins. But chances are that the need for OPU will be a one-off, and then this would be total overkill. Hard to tell what will prove right in the future.